src/internal/testutil/debug.go

```diff
@@ -51,7 +51,7 @@ func DebugFiles(t testing.TB, dataRepo string) (map[string]*globlib.Glob, []stri
 	}
 	for _, app := range []string{"etcd", "pg-bouncer"} {
 		for _, file := range []string{"describe.txt", "logs.txt", "logs-previous**", "logs-loki.txt"} {
-			pattern := path.Join(app, "*", file)
+			pattern := path.Join(app, "**", file)
```

Comment:
what's `**` here?

Reply:
`**` is a glob that consumes all characters, even path separators (`*` stops at path separators).  This is because for the "extra apps" (pgbouncer, etcd, console) we store per-container logs in a directory named after the container now.  Instead of like `etcd/etcd-abcdef-1234/logs.txt` we now use `etcd/etcd-abcdef-1234/etcd/logs.txt`.  This doesn't really matter as everything we install is a single container; just defensive programming if that ever changes.  (Users can do that; Istio and jaeger operator inject sidecar containers.  Before we would just error out, now we'll get those logs too.)

I kind of wanted to do this for pachd as well, but the special cases don't lend themselves to making that particularly easy and it's unlikely to come up.  To make the PR as small as possible I avoided that.  If users ever start using jaeger with an agent sidecar and we want those logs, we'll have to revisit the semantics of `pachctl debug dump --pachd` and decide if we want all containers or just the pachd container ;)